pkg/sfu/receiver.go

```diff
@@ -188,6 +195,10 @@ func (w *WebRTCReceiver) Kind() webrtc.RTPCodecType {
 	return w.kind
 }
 
+func (w *WebRTCReceiver) OnAudioLevel(fn AudioLevelHandle) {
```

Comment:
Routing `onAudioLevel` via `sfu.Receiver` so that we can add it `sfu.TrackReceiver` interface and use it across cloud also.

Reply:
With speaker updates, I know we said that each SFU could perform local computation for remote tracks. I don't think we need to do that anymore. SpeakerUpdates are being forwarded to all media nodes.

Reply:
Yeah, that's true. Okay, let me move all the audio level stuff back into `mediatrack.go`